google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/functions/WindowsPasswordGenerator.java

```diff
@@ -131,34 +131,51 @@ public String apply(AtomicReference<Instance> instance) {
                   operation.get().httpErrorMessage());
          }
 
-         // Retrieve the result from the last line of serialPortOutput; expect JSON string
+          // TODO remove me
+          try {
+              Thread.sleep(3 * 60 * 1000);
+          } catch (InterruptedException e) {
+              Throwables.propagate(e);
+          }
+
+          // Retrieve the result from the last line of serialPortOutput; expect JSON string
          // FIXME should it be "\\n", as Valentin did?
-         Instance.SerialPortOutput serialPortOutput = instanceApi.getSerialPortOutput(instance.get().id(), 4);
-         String entry = Iterables.getLast(Splitter.on("\n").split(serialPortOutput.contents()));
```

Comment:
Splitter.on didn't split the string.
I prefer the more verbose and simpler version of splitting.